src/Controller/Api/TurnkeyProjectController.php

```diff
@@ -13,59 +13,62 @@
 use Symfony\Component\HttpFoundation\JsonResponse;
 use Symfony\Component\HttpFoundation\Response;
 
+/**
+ * @Route("/turnkey-projects")
+ */
 class TurnkeyProjectController extends Controller
 {
     /**
-     * @Route("/turnkey-project/is-pinned", name="api_pinned_turnkey_project")
+     * @Route(name="api_approved_turnkey_projects")
      * @Method("GET")
      */
-    public function getPinnedTurnkeyProjectAction(TurnkeyProjectRepository $repository, Serializer $serializer): Response
+    public function getApprovedTurnkeyProjectAction(TurnkeyProjectRepository $repository, Serializer $serializer): Response
     {
-        $turnkeyProject = $repository->findPinned();
-
         return new JsonResponse(
-            $serializer->serialize($turnkeyProject, 'json', SerializationContext::create()->setGroups(['turnkey_project_read'])),
+            $serializer->serialize($repository->findApprovedOrdered(), 'json', SerializationContext::create()->setGroups(['turnkey_project_list'])),
             JsonResponse::HTTP_OK,
             [],
             true
         );
     }
 
     /**
-     * @Route("/turnkey-project/{slug}", name="api_turnkey_project")
-     * @Entity("turnkeyProject", expr="repository.findOneApprovedBySlug(slug)")
+     * @Route("/count", name="api_count_approved_turnkey_projects")
      * @Method("GET")
      */
-    public function getTurnkeyProjectAction(TurnkeyProject $turnkeyProject, Serializer $serializer): Response
+    public function countApprovedTurnkeyProjectAction(TurnkeyProjectRepository $repository): Response
     {
         return new JsonResponse(
-            $serializer->serialize($turnkeyProject, 'json', SerializationContext::create()->setGroups(['turnkey_project_read'])),
-            JsonResponse::HTTP_OK,
-            [],
-            true
+            ['total' => $repository->countApproved()],
+            JsonResponse::HTTP_OK
         );
     }
 
     /**
-     * @Route("/turnkey-projects/count", name="api_count_approved_turnkey_projects")
+     * @Route("/is-pinned", name="api_pinned_turnkey_project")
```

Comment:
1) what about just `/pinned`?
2) `api_pinned_turnkey_project` => `api_pinned_turnkey_projectS` 
3) is the list of pinned projects ordered?

Reply:
1. Why not `is-pinned`?
2. No, there is only one pinned turnkey project, it's not possible to have more than one.
3. Same that 2.

Reply:
I agree with the naming `/pinned`, it is more in accordance with the action name, and `is-pinned` make us think boolean logic.